recipes/bzip2/all/conanfile.py

```diff
@@ -107,11 +111,9 @@ def package_info(self):
         self.cpp_info.set_property("cmake_build_modules", [self._module_file_rel_path])
         self.cpp_info.libs = ["bz2"]
 
-        self.cpp_info.names["cmake_find_package"] = "BZip2"
-        self.cpp_info.names["cmake_find_package_multi"] = "BZip2"
-        self.cpp_info.build_modules["cmake_find_package"] = [self._module_file_rel_path]
-
-        if self.options.build_executable:
-            bin_path = os.path.join(self.package_folder, "bin")
-            self.output.info("Appending PATH environment variable: {}".format(bin_path))
-            self.env_info.PATH.append(bin_path)
+        if Version(conan_version).major < 2:
```

Comment:
```suggestion
```

Reply:
@jwillikers @toge Since you are also helping to opening/reviewing PRs  -- please mark sure this is not being used ❤️

Reply:
Why? It's a better way that tones of TOTO to track conan v1 stuff which are unused in conan v2 client. Code is better than comments.

Reply:
Conan 2.0 is still beta, which means, not totally stable. It will accept env_info soon, so it should work for both versions. Also, adding an `if` adds more complexity, okay a recipe is small, but still, a new condition for something that should work is too much.

Reply:
@prince-chrismc What about its use in the `header-only` template? https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/header_only/all/conanfile.py

Reply:
https://github.com/conan-io/conan-center-index/pull/13754

Removing it . Thanks for pointing it out